packages/app/components/pack/PackDetails.tsx

```diff
@@ -63,13 +70,13 @@ export function PackDetails() {
       <View
         style={[
           styles.mainContainer,
-          Platform.OS == 'web'
-            ? { minHeight: '100vh' }
-            : { minHeight: Dimensions.get('screen').height },
+          // Platform.OS == 'web'
+          //   ? { minHeight: '100vh' }
+          //   : { minHeight: Dimensions.get('screen').height },
         ]}
       >
-        {!isError && (
-          <>
+        <ScrollView showsVerticalScrollIndicator={false}>
```

Comment:
this causes weird double scroll effect on web, we should avoiding nesting them

Reply:
Yes but this was meant to make the chat opener float inside the view. or may  be I will use fixed for web instead.